contracts/lib/BridgeAssetOperations.sol

```diff
@@ -66,10 +65,31 @@ library BridgeAssetOperations {
         if (tokenInfo.originNetwork == 0) {
             revert TokenNotForkable();
         }
+        (bool successNameCall, bytes memory name) = token.staticcall(
+            /// encoding the function signature of N
+            abi.encodeWithSignature("name()")
+        );
+        if (!successNameCall) {
```

Comment:
Is there a reason for our fallbacks to be named subtly differently from the way upstream does it? 

https://github.com/0xPolygonHermez/zkevm-contracts/blob/d70266b8742672d59d4060019538d03fe0aac181/contracts/PolygonZkEVMBridge.sol#L800